app/assets/javascripts/helpers/customization_helpers.js

```diff
@@ -0,0 +1,17 @@
+function shouldDisplay(element, schoolContext) {
+  switch(element) {
+  case 'house':
+    if (schoolContext && schoolContext.local_id) {
+      return "SHS" === schoolContext.local_id;
+    }
+    return false;
+  case 'counselor': 
+    if (schoolContext && schoolContext.school_type) {
+      return "HS" === schoolContext.school_type;
+    }
+    return false;
+  }
+  return false;
```

Comment:
What is this second layer of `return false` doing? Is it kind of like a default?

Reply:
Yup, forgot about that default thing. :-)